test/repository/contributors/contributors_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:git_hub/core/http/http_exception.dart';
+import 'package:git_hub/models/contributors/contributor_response.dart';
+import 'package:git_hub/repository/contributors/contributors.dart';
+import 'package:mockito/mockito.dart';
+
+import '../../helpers/mock_json_data.dart';
+import '../../mocks/http_service.mocks.dart';
+
+void main() {
+  late ContributorsRepositoryImpl contributorsRepositoryImpl;
+  late MockHttpService mockHttpService;
+  const repositoryFullName = "Alamofire";
+
+  setUp(() {
+    mockHttpService = MockHttpService();
+    contributorsRepositoryImpl = ContributorsRepositoryImpl(httpService: mockHttpService);
+  });
+
+  group("When get contributors is success", () {
+    late List<ContributorResponse> contributors;
+    setUp(() async {
+      final contributorsJson = getMockJson(
```

Comment:
Like I said, we usually don't put commas in the end when it fits in a single line without automatically breaking.